frontend/src/components/Collection/components/CollectionDatasetsGrid/components/Row/DatsetRow/components/MoreDropdown/components/Menu/index.tsx

```diff
@@ -1,86 +1,76 @@
+import DropboxChooser from "src/components/DropboxChooser";
+import { Props } from "src/components/Collection/components/CollectionDatasetsGrid/components/Row/DatsetRow/components/MoreDropdown/components/Menu/types";
+import DeleteDataset from "src/components/Collection/components/CollectionDatasetsGrid/components/Row/DatsetRow/components/DeleteDataset";
 import {
-  MenuItemProps,
-  Intent,
-  Menu as RawMenu,
-  MenuItem,
-} from "@blueprintjs/core";
-import { IconNames } from "@blueprintjs/icons";
-import styled from "@emotion/styled";
-import DropboxChooser, {
-  Props as ChooserProps,
-} from "src/components/DropboxChooser";
-import DeleteDataset from "../../../DeleteDataset";
-import { Collection } from "src/common/entities";
-
-const DeleteButton = (props: MenuItemProps) => {
```

Comment:
Moved `DeleteButton` and `UpdateButton` etc into main component for readability.